docs/reference/inference/put-inference.asciidoc

```diff
@@ -84,6 +84,7 @@ OpenAI service.
 * `azureaistudio`: specify the `completion` or `text_embedding` task type to use the Azure AI Studio service.
 * `elasticsearch`: specify the `text_embedding` task type to use the E5
 built-in model or text embedding models uploaded by Eland.
+* `googleaistudio`: specify the `completion` task to use the Google AI Studio service.
```

Comment:
Would you mind listing `Google AI Studio` also in the list above from line 41 to line 47? I couldn't comment there as that part of the file is unchanged.